projects/saturn/src/test/java/io/fairspace/saturn/vocabulary/VocabulariesTest.java

```diff
@@ -27,11 +30,15 @@ public void setUp() {
 
     @Test
     public void validateVocabulary() {
-        validate(VOCABULARY, SHACL_FOR_SHACL);
+        var dsg = DatasetGraphFactory.createTxnMem();
+        Dataset ds = wrap(dsg);
```

Comment:
this one is already defined (line 24), but not used. One of these can be deleted then